apps/nextjs/src/components/AppComponents/Chat/clear-history.tsx

```diff
@@ -1,81 +1,51 @@
 "use client";
 
 import * as React from "react";
-import { toast } from "react-hot-toast";
 
-import { useRouter } from "next/navigation";
+import { OakBox, OakFlex, OakSpan } from "@oaknational/oak-components";
 
-import {
-  AlertDialog,
-  AlertDialogAction,
-  AlertDialogCancel,
-  AlertDialogContent,
-  AlertDialogDescription,
-  AlertDialogFooter,
-  AlertDialogHeader,
-  AlertDialogTitle,
-  AlertDialogTrigger,
-} from "@/components/AppComponents/Chat/ui/alert-dialog";
-import { Button } from "@/components/AppComponents/Chat/ui/button";
-import { IconSpinner } from "@/components/AppComponents/Chat/ui/icons";
-import { trpc } from "@/utils/trpc";
+import BinIcon from "@/components/BinIcon";
+import OakBoxWithCustomWidth from "@/components/OakBoxWithCustomWidth";
 
-type ClearHistoryProps = Readonly<{
-  isEnabled: boolean;
-}>;
-
-export function ClearHistory({ isEnabled = false }: ClearHistoryProps) {
-  const clearChats = trpc.chat.appSessions.deleteAllChats.useMutation({
-    onSuccess() {
-      toast.success("Chat history cleared");
-      setOpen(false);
-      router.push("/");
-    },
-    onError() {
-      toast.error("Failed to clear chat history");
-    },
-  }).mutate;
+import { useDialog } from "../DialogContext";
 
-  const [open, setOpen] = React.useState(false);
-  const [isPending, startTransition] = React.useTransition();
-  const router = useRouter();
+type ClearHistoryProps = {
+  isEnabled: boolean;
+};
 
+export function ClearHistory({ isEnabled }: Readonly<ClearHistoryProps>) {
+  const { setDialogWindow } = useDialog();
+  if (!isEnabled) {
+    return null;
+  }
   return (
-    <AlertDialog open={open} onOpenChange={setOpen}>
-      <AlertDialogTrigger asChild>
-        <Button
-          variant="ghost"
-          disabled={!isEnabled || isPending}
-          className="fixed bottom-0 right-0 z-40 flex w-full justify-center bg-white p-16 pl-0 sm:w-[298px]"
+    <OakBoxWithCustomWidth
+      width={447}
+      $position="fixed"
```

Comment:
I can't see the designs, but if this is within a container like a sidebar, should it be position:relative?